rise-financial-es6.js

```diff
@@ -253,8 +253,12 @@
         return;
       }
 
-      const financial = this.$.financial,
-        params = this._getParams( instruments, fields );
+      const financial = this.$.financial;
+
+      // set callback with the same value it was set on the responseHandler of the tqx parameter
+      financial.callbackValue = ( btoa( ( this.id ? this.id : "" ) + this._getDataCacheKey() ) ).substr( 0, 10 ) + ( Math.random().toString() ).substring( 2 );
```

Comment:
@rodrigopavezi What is the reason for the additional `+ ( Math.random().toString() ).substring( 2 );` at the end?

Reply:
@stulees so we don't have this issue here https://user-images.githubusercontent.com/8008948/31774523-008098b2-b4b4-11e7-9b34-c223eb433f02.png happening. It was happening because the presentation was requesting data repeatedly with the same key when there is no symbol set. Cheers

Reply:
@rodrigopavezi Okay thanks